app/components/bulk_actions_form_component.rb

```diff
@@ -1,9 +1,12 @@
 # frozen_string_literal: true
 
 class BulkActionsFormComponent < ApplicationComponent
-  def initialize(form:, last_search:)
+  def initialize(form:)
     @form = form
-    @last_search = last_search
+  end
```

Comment:
we don't need the last_search passed in anymore, we can instead use the blacklight search state (which parses from the querystring)